public/gatk-queue/src/main/scala/org/broadinstitute/gatk/queue/QSettings.scala

```diff
@@ -99,6 +100,13 @@ class QSettings {
   @Argument(fullName="log_directory", shortName="logDir", doc="Directory to write log files into.", required=false)
   var logDirectory: File = _
 
+  @Argument(fullName = "quality_of_service", shortName = "qos", doc="Quality of Service for Slurm", required=false)
+  @ClassType(classOf[String])
+  var qualityOfSerice: Option[String] = None
+
+  @Argument(fullName="waitBeforeJob", shortName="waitBeforeJob", doc="For Slurm the job submision waits number of seconds, incase of io delay on storage. Default is 5 seconds", required=false)
```

Comment:
Found a few typos and grammar issues here: 

`submision` -> `submission`
`incase` -> `in case` 
`waits number` -> `waits a number`
Also, all sentences must end in a dot (`.`)